example_mp.py

```diff
@@ -0,0 +1,49 @@
+
+"""
+Basic usage of PPK2 Python API - multiprocessing version
+The basic ampere mode sequence is:
+1. read modifiers
+2. set ampere mode
+3. read stream of data
+"""
+import time
+from ppk2_api.ppk2_api.ppk2_api import PPK2_MP
```

Comment:
I don't know why this is not updated in the original sample, but I think it should be `from src.ppk2_api.ppk2_api import PPK2_API`, agreed?

Reply:
There is one ".ppk2_api" too much which is a typo. I left "from ppk2_api..." instead of "from src...." to match the original example (but in testing used the "from src.ppk2_api"). I would rather leave it to match the original example (with a typo fixed in last commit) and then you can change this to "from src.ppk2_api" on both files